### DISCUSSION TITLE
Docker on Mac

### DISCUSSION THREAD

**Discussion by mauede33** [CCPPETMR/SIRF-SuperBuild#612] [Q&A]
I finally could complete all the steps described in Dr Atkinson's video.
However, my Docker dashboard does not have all the images shown in the video
For instance, I do not have a Gadgetron.
In the Jupyter terminal, I see fewer items (base) sirf$ pwd
/devel
(base) sirf$ ls
CIL-Demos  requirements-test.txt  shared.txt  SIRF-contrib  SIRF-Exercises  test.sh
(base) sirf$ ls -la
total 16
drwxr-xr-x  9 sirfuser dialout  288 Jul  3 14:56 .
drwxr-xr-x  1 root     root    4096 Jul  3 14:55 ..
drwxr-xr-x  3 sirfuser dialout   96 Jul  2 09:47 .ccache
drwxr-xr-x  9 jovyan   users    288 Jul  2 18:12 CIL-Demos
-rw-r--r--  1 sirfuser dialout   14 Jul  2 09:47 requirements-test.txt
-rw-r--r--  1 sirfuser dialout  261 Jul  2 09:47 shared.txt
lrwxr-xr-x  1 sirfuser dialout   48 Jul  3 14:56 SIRF-contrib -> /opt/SIRF-SuperBuild/INSTALL/python/sirf/contrib
drwxr-xr-x 16 jovyan   users    512 Jul  2 18:10 SIRF-Exercises
-rwxr-xr-x  1 sirfuser dialout  787 Jul  2 09:47 test.sh
(base) sirf$

Is that OK?
I can see the 1st-week notebooks when I navigate the Jupyter terminal.
Shall I run the notebooks in the Jupyter terminal?
Thank you

**Comment by KrisThielemans** [comment]
This looks all fine. Now you have to use the Jupyter interface to open one of the notebooks (just like you would do in the VM or the cloud). There is little need for using the terminal (launched from Jupyter), except for downloading the data.

**Reply by DANAJK** [reply]
Although there is a Gadgetron docker on my Mac (I assume this is visible in the video), it is old and I don't think it is relevant to the SIRF course. My understanding is that the SIRF Docker container will contain everything and you do not need to download any other Dockers.

**Reply by DANAJK** [reply]
With regards to Docker daemons on a Mac: with Docker Desktop running, I think that is sufficient. You can tell if it is running by left-clicking on the Docker whale icon in the top bar. The first menu entry tells you its status.
